pkg/graphviz/traceflow.go

```diff
@@ -475,6 +484,67 @@ func GenGraph(tf *crdv1alpha1.Traceflow) (string, error) {
 		}
 		edge.Attrs[gographviz.Constraint] = "false"
 	}
-
+	if tf.Spec.LiveTraffic && tf.Status.Phase == crdv1alpha1.Succeeded {
+		err = createCapturedPacketNode(graph, cluster2.Name, tf)
+		if err != nil {
+			return "", err
+		}
+	}
 	return genOutput(graph, false), nil
 }
+
+func getCapturedPacketLabel(tf *crdv1alpha1.Traceflow) string {
+	label := "Captured Packet:\\lDestination IP:" + tf.Status.CapturedPacket.DstIP + "\\l"
+
+	if tf.Status.CapturedPacket.IPHeader != (crdv1alpha1.IPHeader{}) {
+		label = label + "IPv4 Header: " + "\\l" +
+			"    Flags: " + fmt.Sprintf("%d", tf.Status.CapturedPacket.IPHeader.Flags) + "\\l" +
+			"    Protocol: " + fmt.Sprintf("%d", tf.Status.CapturedPacket.IPHeader.Protocol) + "\\l" +
+			"    TTL: " + fmt.Sprintf("%d", tf.Status.CapturedPacket.IPHeader.TTL) + "\\l"
+	}
+
+	if tf.Status.CapturedPacket.IPv6Header != nil {
+		label = label + "IPv6 Header: \\l" + "    Next Header: " +
+			fmt.Sprintf("%d", *tf.Status.CapturedPacket.IPv6Header.NextHeader) + "\\l" +
+			"    Hop Limit: " + fmt.Sprintf("%d", tf.Status.CapturedPacket.IPv6Header.HopLimit) + "\\l"
+	}
+
+	label = label + "Length: " + fmt.Sprintf("%d", tf.Status.CapturedPacket.Length) + "\\l" +
```

Comment:
I feel better to order the fields as:
Destination IP
Source IP
Length
IP Header
TransportHeader

Reply:
done